blueprints/page-object-helper/files/tests/pages/helpers/__name__.js

```diff
@@ -0,0 +1,7 @@
+import PageObject from '../../page-object';
+
+function checked(selector /*, options */) {
+  return $(selector).prop('checked');
+}
+
+export default PageObject.customHelper(checked);
```

Comment:
We can use the name provided by the user as the name of the function:

```js
function <%= camelizedModuleName %>(selector/*, options*/) {
  return ...;
}

export default PageObject.customHelper(<%= camelizedModuleName %>);
```

What do you think?

http://www.ember-cli.com/extending/#template-variables-aka-locals